pytorch_lightning/loops/epoch/training_epoch_loop.py

```diff
@@ -221,6 +221,8 @@ def on_run_end(self) -> List[List[STEP_OUTPUT]]:
         self.trainer.call_hook('on_epoch_end')
         self.trainer.logger_connector.on_epoch_end()
 
+        self.update_lr_schedulers('epoch', update_plateau_schedulers=True)
```

Comment:
also note to reviewer: by moving this it will allow us to add the lr_scheduler progress tracking in #8362 into one single place for easier access!